Features apple page

kiptubei opened the pull request:
Created the apple welcome page as discussed in the Readme file.
Features include : 

- Background image
- Applying linear gradients to the navbar.
- Align elements using flex

kiptubei committed:
Added stickler styleint.config.js created project structure

kiptubei committed:
Edited gitignore file excluded stylelint.config.js for Atom IDE compatibility

kiptubei committed:
Added stickler styleint.config.js created project structure

kiptubei committed:
Edited gitignore file excluded stylelint.config.js for Atom IDE compatibility

kiptubei committed:
Create general UI structure,background image div,banner section,footer and copyright section

kiptubei committed:
Add apple styling,background image and gradient to nav-bar

kiptubei committed:
Added readme sile and screenshot

kiptubei committed:
Added rawgithack link ,fixed html validator errors

kiptubei committed:
Changed font to Manrope to meore closely match original

kiptubei committed:
Added raw git hack link to live page

adaorachi commented on index.html:
```diff
@@ -0,0 +1,130 @@
+          </nav>
+        </header>
+        <section class="banner container">
+          <p class="catch-phrase">What will your verse be?</p>
+          <h3>
+            Everyone has something to share. See how people are using <br />
+            iPad in their unique ways to add to the world's story.
+          </h3>
```
- [**optional**] Here, I noticed a misuse of semantic HTML elements. The element in the `<p>` tag is considered to be the title of the site and so should have a header tag and which is higher than the header tag `<h3>` you gave to the subtitle. So, you should consider giving it an appropriate semantic tag.

adaorachi replied:
-[**optional**] Here, I noticed a misuse of semantic HTML elements. The element in the `<p>` tag is considered to be the title of the site and so should have a header tag and which is higher than the header tag `<h3>`  you gave to the subtitle. So, you should consider giving it an appropriate semantic tag.

adaorachi commented on index.html:
```diff
@@ -0,0 +1,130 @@
+        <header class="primary-header container">
+          <nav class="nav primary-nav container">
+            <div>
+              <button type="button" class="btn-apple">
+                <!--Apple-->
+                <img
+                  src="images/apple-logo-png.png"
+                  alt="apple-logo"
+                />&nbsp;</button
+              ><!--
+            --><button type="button" class="btn-apple">Store</button
+              ><!--
+            --><button type="button" class="btn-apple">Mac</button
+              ><!--
+            --><button type="button" class="btn-apple">iPod</button
+              ><!--
+            --><button type="button" class="btn-apple">iPhone</button
+              ><!--
+            --><button type="button" class="btn-apple">iPad</button
+              ><!--6.5
+            --><button type="button" class="btn-apple">iTunes</button
+              ><!--
+            --><button type="button" class="btn-apple">Support</button
+              ><!--
```
- [ ] Here, the elements should be links ( `<a>` tag) that have `href` attributes linking to other sections of the page or just a '#'. I would also encourage you to make use of suitable semantic HTML elements like `<li>` instead of the `<button>` tag like you have it. It will make your codes look cleaner and modular.

adaorachi commented:
👋  Hi @kiptubei,

Well done on the completion of the first draft of this project. You did really well on it. Your images and contents are well aligned and positioned. You have a well written Pull Request summary and commit messages. In all, your project's view looks very nice! 👍💯🌟 

However, there are a couple of changes you'll need to make to get approval for this project.
So, hang in there, you are just a step away from completing this project.

---
### **CHANGES REQUIRED** ♻️ 
---

adaorachi commented on index.html:
```diff
@@ -0,0 +1,130 @@
+        <header class="primary-header container">
+          <nav class="nav primary-nav container">
+            <div>
+              <button type="button" class="btn-apple">
+                <!--Apple-->
+                <img
+                  src="images/apple-logo-png.png"
+                  alt="apple-logo"
+                />&nbsp;</button
+              ><!--
+            --><button type="button" class="btn-apple">Store</button
+              ><!--
+            --><button type="button" class="btn-apple">Mac</button
+              ><!--
+            --><button type="button" class="btn-apple">iPod</button
+              ><!--
+            --><button type="button" class="btn-apple">iPhone</button
+              ><!--
+            --><button type="button" class="btn-apple">iPad</button
+              ><!--6.5
+            --><button type="button" class="btn-apple">iTunes</button
+              ><!--
+            --><button type="button" class="btn-apple">Support</button
+              ><!--
+            --></div>
```
- [ ] Here, the elements should be links ( `<a>` tag) that have `href `attributes linking to other sections of the page or just a '#'. I would also encourage you to make use of suitable semantic HTML elements like `<li>` instead of the `<button>` tag like you have it. It will make your codes look cleaner and modular.

adaorachi commented on index.html:
```diff
@@ -0,0 +1,130 @@
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Styles Conference</title>
```
-[**optional**] You should consider giving an appropriate title to your project. Preferably a name that briefly describes your project.

adaorachi commented:
**The following requirements are considered OPTIONAL and so, are NOT mandatory but I will encourage you to take a moment to look into it as it will be helpful to you in this project and future ones.
NOTE: Requirements NOT marked as OPTIONAL are mandatory!**

---

- [**optional**] You should also consider adding a hover effect when the navigation links are hovered on. This will improve your project aesthetic look.

---

- [**optional**] On the footer, the colors of your content are quite dim and not very easy to read. Consider using darker color shades for the text and links.

adaorachi commented:
### **After implementing the MANDATORY changes highlighted above, request for another review by submitting a Code Review Form.**

---

Should you have any questions or need any help, please feel free to reach out to me on Slack with the username (@adaorachi).

Happy Coding and Good-luck on it. ☺️
And keep being an awesome developer! 👍 

**MaryAnn Chukwuka**

kiptubei committed:
Added appropriate title,edited use of semantic tags to conform to standard,changed use of button to use li and a href tags,made footer text bold inorder to be seen clearly

kiptubei commented:
- Added appropriate title,
- edited use of semantic tags to conform to the standard,
- changed use of button to use li and a href tags,
- made footer text bold in-order to be seen clearly

kiptubei committed:
Update README.md

kiptubei committed:
Readme edit

Edited live link

kiptubei committed:
Edited readme live link

kiptubei committed:
Merge branch 'features-apple-page' of https://github.com/kiptubei/Apple_Background into features-apple-page
Insert TSE recommendations

kiptubei committed:
removed linter errors

kiptubei committed:
Fixed syntax issue with the search bar

kiptubei committed:
Readme Live Link

Added raw githack live link

uryelah commented:
Hello **Mark**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all great job with the styling of this project it looks a lot like the original page, the navbar is exceptionally well made.🌟**

There is only one last issue you need to fix order to go to the next project, so read the points below carefully and keep up the good job 💪

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] Make sure there are no errors or warnings in your HTML validation. It looks like a new one appeared after applying the changes from the last review.

-----

- [ ] [OPTIONAL] The logo you took a screenshot of has a different background from your navbar and also a lower resolution. Use a real apple icon and then style it or use an image with a transparent background for a more professional looking project.

----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

kiptubei committed:
Fixed html validator issue

kiptubei commented:
- removed HTML validator errors

sumancrest0001 commented:
**Congratulation PROJECT APPROVED**
**Awesome WORK @kiptubei  :clap: :clap:**  
It looks perfect to me and it's a Yes from my side. Now, you can merge your branch to master and submit the project completion form. But before that, I  suggest you make a change to the apple logo stated by the previous TSE. It will make your site a clone of the original site.
**Happy coding !**
![mission](https://user-images.githubusercontent.com/37747619/79842129-2549f580-83d8-11ea-9500-0569e84a7c44.gif)
